src/main/resources/outputSchema.avsc

```diff
@@ -0,0 +1,103 @@
+{
+   "namespace":"com.zeotap.daap.mdm.avro.schema",
+   "type":"record",
+   "name":"MDM",
+   "fields":[
+      {
+         "name":"Identifier",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"DpId",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Gender",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"Device",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"Age",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Zipcode",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"Language",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"Gender_dpid",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Device_dpid",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Age_dpid",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Zipcode_dpid",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      },
+      {
+         "name":"Language_dpid",
+         "type":[
+            "null",
+            "int"
+         ],
+         "default":null
+      }
+   ]
+}
```

Comment:
Add new line here. Make sure that the newline setting in IntelliJ is turned on

Reply:
Added new line. The newline setting in IntelliJ somehow doesn't add newline to every file type

Reply:
It just doesn't add to already existing files. Any new file is created with this setting